WordPress/src/main/res/values/strings.xml

```diff
@@ -1395,7 +1395,7 @@
     <!-- Error Messages -->
     <!-- The following messages can\'t be factorized due to i18n -->
     <string name="error_refresh_posts">Posts couldn\'t be refreshed at this time</string>
-    <string name="error_refresh_pages">Pages couldn\'t be refreshed at this time</string>
+    <string name="error_refresh_pages" tools:ignore="UnusedResources">Pages couldn\'t be refreshed at this time</string>
```

Comment:
If this is unused can we just get rid of the string res?

Reply:
It is not unused, but lint is falsely failing with an unused resources error. I don't know of another way to fix the error.

https://github.com/wordpress-mobile/WordPress-Android/blob/ea8da2312151a3ded7ca4bf3b566ef4d38e10e7c/WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt#L184